setup.py

```diff
@@ -33,7 +33,7 @@ def find_stub_files(name: str) -> List[str]:
 
 # Keep compatible-mypy major.minor version pinned to what we use in CI (requirements.txt)
 extras_require = {
-    "compatible-mypy": ["mypy==1.5.*"],
+    "compatible-mypy": ["mypy==1.6.*"],
```

Comment:
As suggested by flaeppe

```suggestion
    "compatible-mypy": ["mypy~=1.6.0"],
```